### ISSUE TITLE
 [coreml] MLModelAsset: load failed with error

### ISSUE THREAD

**Issue by aloveric** [mdramos/fast-style-transfer-coreml#1]
Import your CoreML and run iOS code, it has this issue:

<img width="1020" alt="screen shot 2018-01-21 at 10 38 18 pm" src="https://user-images.githubusercontent.com/8198881/35195268-e3531bec-fefb-11e7-9c78-7e3dea10bd32.png">

**Comment by aloveric** [comment]
BTW, Simulation works fine, error on real iPhone

**Comment by backnotprop** [comment]
I’ll give it a check, what is the iOS version on the phone?

**Comment by aloveric** [comment]
@mdramos Thanks for your quick feedback. It is iPhone 6 and iOS 11.1.2. Also check with iPad Air 2 on iOS 11.1.2 with same issue.

**Comment by backnotprop** [comment]
related: https://forums.developer.apple.com/thread/82700

**Comment by aloveric** [comment]
Also found this but there is no solution to fix it ...

**Comment by jst000000** [comment]
i run the code on my iphoneX but its not work well but good with iphone7P and Simulation

**Comment by backnotprop** [comment]
I'm getting better results with lazy loading the models separately, still experiencing some memory issues, so I will continue to look for a better solution.

The first step in the right direction definitely seems to be lazy loading.

**Comment by backnotprop** [comment]
actually, a single model in memory seems to work best without issues. Just nil-ing out old models and keeping one in memory.

Ill push this fix in a bit, but will hopefully find a better solution for having multiple models loaded in memory.

**Comment by jst000000** [comment]
yes, most of phone work well with two models but break with three

**Comment by aloveric** [comment]
<img width="760" alt="screen shot 2018-02-01 at 5 42 15 pm" src="https://user-images.githubusercontent.com/8198881/35671737-4ea8b3ce-0777-11e8-82b5-ca5723b5a592.png">

Guys, I left only one model and it will get whole black screen with loading then show the lock screen via iPhone 6 and iOS 11.1.2

**Comment by mithileshggn** [comment]
Running on simulator only but not in real device? please suggest why? Even though device is running with iOS 11.0

**Comment by bbthitenp** [comment]
![Screenshot 2020-02-13 at 2 59 38 PM](https://user-images.githubusercontent.com/57610629/74420648-cd3fc180-4e71-11ea-9c62-6bd704a73996.png)

Way not run in Real Device?

**Comment by backnotprop** [comment]
Hey all, I see folks stay actively commenting here. 

The models this produce are most likely too large for devices. A new architecture needs to be slimmed down and there have been many advancements/examples to do this.

Im going to archive the repo, as I can not maintain and this should be used as experimentation only.
